endoreg_db/services/pdf_import.py

```
@@ -747,73 +842,142 @@ def _cleanup_on_error(self):
+                else:
+                    sensitive_dir = pdf_dir / "sensitive" if pdf_dir else None
+                    sensitive_count = len(list(sensitive_dir.glob("*"))) if sensitive_dir and sensitive_dir.exists() else None
+
+                logger.info(
+                    "PDF import error cleanup counts - raw: %s, pdf: %s, sensitive: %s",
+                    raw_count,
+                    pdf_count,
+                    sensitive_count,
+                )
+            except Exception:
+                pass
```

Pylint (4.0.2)

[refactor] Too many nested blocks (6/5)

(R1702)